main.go

```diff
@@ -131,32 +131,30 @@ print it with "mkcert -CAROOT".
 	m.makeCert(args)
 }
 
+func getCAROOTFolder() string {
+	if runtime.GOOS == "windows" {
+		return os.Getenv("LocalAppData")
+	}
+	dir := os.Getenv("XDG_DATA_HOME")
+	if dir != "" {
+		return dir
+	}
+	home := os.Getenv("HOME")
+	if home == "" {
+		return ""
```

Comment:
Just realized that this needs to return and empty string from the calling function, too.

How would you feel about introducing an error return value instead of using the empty string as a sentinel?

Reply:
I just wouldn't split it into two functions. You can just have a `case os.Getenv("XDG_DATA_HOME") != ""` above `case runtime.GOOS == "darwin"`.

Reply:
Wait, you can? 🤯

Reply:
Wait, you can? 🤯

Reply:
Done!